js_modules/dagster-ui/packages/ui-core/src/automation/VirtualizedAutomationScheduleRow.tsx

```diff
@@ -133,22 +135,32 @@ export const VirtualizedAutomationScheduleRow = forwardRef(
             </Tooltip>
           </RowCell>
           <RowCell>
+            {/* Left aligned content */}
```

Comment:
chatgpt comment 🧐 ?

Reply:
Yeah it was but I wanted to keep it to show what I was doing

Reply:
Nice - it actually wouldn't be terrible long term for us to add an `align: 'left'` option to Box that is a short-hand for `alignItems: flex-start` -- it's always a bit awkward.

Fun fact, if you are using a right-to-left language like arabic, flex-start is actually right aligned automatically unless you explicitly set `dir="ltr"`